packages/apps/shopify-app-react-router/src/server/authenticate/admin/helpers/validate-shop-and-host-params.ts

```diff
@@ -1,14 +1,14 @@
 import {redirect} from 'react-router';
 
-import {BasicParams} from '../../../types';
+import {BasicParams, AppDistribution} from '../../../types';
 
 export function validateShopAndHostParams(
   params: BasicParams,
   request: Request,
 ) {
   const {api, config, logger} = params;
 
-  if (config.isEmbeddedApp) {
+  if (config.distribution !== AppDistribution.ShopifyAdmin) {
```

Comment:
With the amount of checks I'm seeing like this, I wonder if we centralize this `isEmbedded(config.distribution)`?

Not how I feel about that suggestion, so feel free to ignore